src/Consolonia.Gallery/View/MainWindow.axaml

```diff
@@ -7,6 +7,7 @@
         d:DesignWidth="800"
         d:DesignHeight="450"
```

Comment:
Bottom shadow of the dialog is lost 
<img width="1494" height="1000" alt="image" src="https://github.com/user-attachments/assets/83b4eb2e-5663-4d58-bf9f-c59f7ddf9ce8" />